examples/CQL Query/useCaseUsingCassandraQueryLang.cfg

```diff
@@ -0,0 +1,34 @@
+PATROL_CONFIG
+"/SENTRY/X/useCaseUsingCassandraQueryLang/actionList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/collectionErrorCountAutoAcknowledgeTime" = { REPLACE = "900" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/databaseType" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/disabled" = { REPLACE = "0" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/embeddedFileList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/label" = { REPLACE = "Use Case - Using Cassandra Query Language via Command Line" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/macroList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/messageList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/parameterList" = { REPLACE = "CollectionErrorCount;1" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/productVersion" = { REPLACE = "10300" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/runAlertForEachCollectionError" = { REPLACE = "1" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/shareCookies" = { REPLACE = "1" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/systemType" = { REPLACE = "other" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/thresholdList" = { REPLACE = "CollectionErrorCount;0 0 0 0 0 0;1 1 2147483646 0 0 2;0 0 0 0 0 0" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang/type" = { REPLACE = "Template" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/actionList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/credentials" = { REPLACE = "System Credentials" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/exitCodeStatus" = { REPLACE = "0" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/exitCodes" = { REPLACE = "0" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/label" = { REPLACE = "CQL Query Using cqlsh" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/messageList" = { REPLACE = "" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/neverEnding" = { REPLACE = "0" },
+"/SENTRY/X/useCaseUsingCassandraQueryLang:cqlQueryUsingCqlsh/osCommand" = { REPLACE = "cqlsh -u %{USERNAME} -p %{PASSWORD} \"SELECT * FROM tablename WHERE column='condition'\"" },
```

Comment:
The following command line giving in your example is wrong

You have to add "-e" in your commandline

```
cqlsh -h
  -u USERNAME, --username=USERNAME
                        Authenticate as user.
  -p PASSWORD, --password=PASSWORD
                        Authenticate using password.
  -k KEYSPACE, --keyspace=KEYSPACE
                        Authenticate to the given keyspace.
  -f FILE, --file=FILE  Execute commands from FILE, then exit
  --debug               Show additional debugging information
  --coverage            Collect coverage data
  --encoding=ENCODING   Specify a non-default encoding for output. (Default:
                        utf-8)
  --cqlshrc=CQLSHRC     Specify an alternative cqlshrc file location.
  --cqlversion=CQLVERSION
                        Specify a particular CQL version, by default the
                        highest version supported by the server will be used.
                        Examples: "3.0.3", "3.1.0"
  --protocol-version=PROTOCOL_VERSION
                        Specify a specific protcol version otherwise the
                        client will default and downgrade as necessary
  -e EXECUTE, --execute=EXECUTE
                        Execute the statement and quit.
```